siac/main.go

```diff
@@ -29,8 +28,13 @@ func apiGet(call string) (*http.Response, error) {
 	if host, port, _ := net.SplitHostPort(addr); host == "" {
 		addr = net.JoinHostPort("localhost", port)
 	}
-
-	resp, err := http.Get("http://" + addr + call)
+	client := &http.Client{}
+	req, err := http.NewRequest("GET", "http://"+addr+call, nil)
+	if err != nil {
+		return nil, errors.New("no response from daemon")
```

Comment:
`NewRequest` only returns an error if the url is malformed, and we know in advance that that will never be the case. So this error (and its parallel in `apiPost`) doesn't need to be checked.